### PR TITLE
Adds collection of tabs to trustyle

### PR DESCRIPTION
**Pull request by NathanHynes**
# Description

adds tabs to trustyle 

tab-gif

# Checklist

Pull request contains:

- [x] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [x] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [x] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

**Comment by callumacrae** [comment]
Does this have a horizontal scrollbar when using a mouse instead of a trackpad and on windows?

**Comment by NathanHynes** [comment]
> Does this have a horizontal scrollbar when using a mouse instead of a trackpad and on windows?

@callumacrae I believe currently this will only support scrolling with a trackpad. I would be keen to know if you have any ideas on how this can support click and drag to scroll

**Comment by callumacrae** [comment]
@NathanHynes in the past when i've had to deal with tabs it's required a design change.

One approach would be to move the off-screen tabs into a "more" dropdown, like this:

image

image

However this can be difficult to do responsively and efficiently. In the past I've solved it by allowing the links to appear off screen during page load (and when JS is disabled) like how it is at the moment, and then after the page has loaded incrementally remove items until they all fit on screen. There might be a better way though!

File: src/themes/money/theme.json [hunk 1]
```diff
@@ -581,6 +581,33 @@
       }
     },
 
+    "collectionTabs": {
+  "base":{
```
Comment (callumacrae) [inline src/themes/money/theme.json:585]: tiny thing - indentation here is a bit off

File: src/themes/uswitch/theme.json [hunk 2]
```diff
@@ -505,6 +505,33 @@
       }
     },
 
+    "collectionTabs": {
+  "base":{
```
Comment (callumacrae) [inline src/themes/uswitch/theme.json:509]: same here

File: src/compounds/tabs/.npmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (jonathan-fielding) [inline src/compounds/tabs/.npmrc:1]: why would we not have a package.lock, is this a pattern on this repo?

File: src/compounds/tabs/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "clean": "rm -rf lib",
+    "build": "npm run clean && tsc"
+  },
+  "peerDependencies": {
+    "@emotion/core": "^10.0.27",
+    "react": "^16.7.0"
+  },
+  "dependencies": {
+    "@uswitch/trustyle.flex-grid": "^2.2.2",
+    "react-indiana-drag-scroll": "^1.6.1"
```
Comment (jonathan-fielding) [inline src/compounds/tabs/package.json:21]: the browser will need to parse 8kb of JS for this, is there any smaller libraries

File: src/compounds/tabs/.npmrc (hunk 3 above)
Comment (NathanHynes) [reply src/compounds/tabs/.npmrc:1]: I believe this is a pattern, check the `_element-template`

Comment (callumacrae) [reply src/compounds/tabs/.npmrc:1]: Libraries don't need lockfiles - they were causing issues with the build (remember the double PRs and separate commits with package locks after publishing?) and they're not used when the package is installed anyway.

File: src/compounds/tabs/package.json (hunk 4 above)
Comment (callumacrae) [reply src/compounds/tabs/package.json:21]: Where did you get that number from? It seems to be 4.7kb before minifying: https://unpkg.com/react-indiana-drag-scroll@1.6.1/dist/index.js

**Pull request merged**